main/guides/orchestration/getting-started/contract-walkthrough.md

```diff
@@ -0,0 +1,5 @@
+# dApp Orchestration Basics
```

Comment:
_question_

I might be missing some context here, did we plan to make `dapp-orchestration-basics` self-documenting and that's why this page just redirects to the actual repo?

Reply:
good q @LuqiPan this will be a contract walkthrough. self documenting is a good idea 👀